front/app/containers/ProjectsShowPage/timeline/PhaseNavigation.tsx

```diff
@@ -116,15 +132,18 @@ const PhaseNavigation = memo<Props & WrappedComponentProps>(
         const prevPhase = phases.data[prevPhaseIndex];
         selectPhase(prevPhase);
       }
-    }, [phases, selectedPhase]);
+    }, [phases, selectedPhase, selectPhase]);
 
     const goToCurrentPhase = useCallback(() => {
       if (phases) {
         trackEventByName(tracks.clickCurrentPhaseButton);
         const currentPhase = getCurrentPhase(phases.data);
-        selectPhase(currentPhase);
+
+        if (currentPhase) {
+          selectPhase(currentPhase);
+        }
       }
-    }, [phases]);
+    }, [phases, selectPhase]);
```

Comment:
Here too, I think we can delete all useCallback instances. It'll make the code cleaner.